platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -155,16 +155,39 @@ public void setView(final GodotRenderView view) {
 	// ===========================================================
 	@Override
 	public boolean onKeyDown(final int keyCode, final KeyEvent keyEvent) {
-		super.onKeyDown(keyCode, keyEvent);
+		// pass event to godot in special cases
+		if (needHandlingInGodot(keyCode, keyEvent)) {
+			mRenderView.getView().onKeyDown(keyCode, keyEvent);
```

Comment:
Here and below (in `onKeyUp`) as mentioned by @pouleyKetchoupp use `GodotInputHandler` directly instead.
Follow the pattern used in the `RenderView` instances. For example:
```
if (needHandlingInGodot(...) && inputHandler.onKeyDown(...)) return true else return super.onKeyDown(...)
```

This way you still get the side effect you were referring to when the `GodotInputHandler` doesn't handle the event.

To facilitate, you should move the `KeyEvent.KEYCODE_BACK` logic below at the top of the method.

Reply:
The problem is that `GodotInputManager` is not singleton and EditText can't get it from anywhere (it's created within Godot which also doesn't have it), and creating a new `GodotInputManager` have worse side effects (double listening)

there is 2 options, 1 staying like that, or 2. adding `getInputManager` to `SurfaceView`

Reply:
For now, use option `2` and add a `getInputHandler` to the `RenderView` interface. 
I'll clean it up once I fix the dependencies between `GodotInputHandler` and the `RenderView` implementations.

Reply:
Ok, ill have time for that tomorrow

Reply:
Done